docs/streams/developer-guide/config-streams.html

```diff
@@ -261,10 +261,10 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
           </tr>
           <tr class="row-odd"><td>statestore.cache.max.bytes</td>
             <td>Medium</td>
-            <td colspan="2">Maximum number of memory bytes to be used for record caches across all threads.</td>
+            <td colspan="2">Maximum number of memory bytes to be used for record caches across all threads. Note that at the debug level you can use <code>cache.size</code> to monitor the actual size of the Kafka Streams cache.</td>
```

Comment:
Not clear to me, what the added sentence means? Are you referring to JMX metrics?